fs/sas-frontend-app
# Turned off react/react-in-jsx-scope rule

## Summary

Turned off `react/react-in-jsx-scope` rule

### List of steps to manually test introduced functionality:

-   Open `src/App.jsx`
-   Check if it has `import React from "react"`
